packages/synthetic-chain/src/lib/commonUpgradeHelpers.ts

```diff
@@ -167,6 +167,25 @@ export const addUser = async (user: string) => {
   return getUser(user);
 };
 
+interface V047ProposalMessage {
```

Comment:
can these be imported from somewhere canonical?

if not, please at least leave comments referencing how these types are derived